series-tmdb.en.yaml

```diff
@@ -2888,6 +2901,23 @@ entries:
       - season: 5
         anilist-id: 142481
 
+  - title: "Yu-Gi-Oh! Duel Monsters"
+    guid: plex://show/5d9c081dba6eb9001fb9c35c
+    # imdb: https://www.imdb.com/title/tt0247902/
+    # tmdb: https://www.themoviedb.org/tv/902
+    # tvdb: https://www.thetvdb.com/dereferrer/series/76894
+    seasons:
+      - season: 1
+        anilist-id: 481
+      - season: 2
```

Comment:
Yugioh on TMDB only has 1 season

Reply:
I've removed the additional season entries